rero_ils/modules/items/api.py

```diff
@@ -772,11 +771,37 @@ def checkin(self, current_loan, **kwargs):
             item, validate_action = self.validate_request(**request)
         return self, actions
 
+    def prior_checkout_actions(self, data):
```

Comment:
data can be renamed as action_params if you want

Reply:
Done.